libnopegl/src/rendertarget.c

```diff
@@ -38,6 +39,10 @@ int ngli_rendertarget_init(struct rendertarget *s, const struct rendertarget_par
     ngli_assert(params->nb_colors <= NGLI_MAX_COLOR_ATTACHMENTS);
     ngli_assert(params->nb_colors <= limits->max_color_attachments);
 
+    if (params->depth_stencil.resolve_target) {
+        ngli_assert(features & NGLI_FEATURE_DEPTH_STENCIL_RESOLVE);
```

Comment:
There is a features parameter exposed to the rtt node: I'm assuming it can't cause this assert to be triggered?

Reply:
Yes, see:
``` c
        if (!(gpu_ctx->features & NGLI_FEATURE_DEPTH_STENCIL_RESOLVE) && o->samples > 0) {
            LOG(ERROR, "context does not support resolving depth/stencil attachments");
            return NGL_ERROR_GRAPHICS_UNSUPPORTED;
        }
```